src/OrleansBenchmarks/MapReduce/MapReduceBenchmark.cs

```diff
@@ -0,0 +1,175 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using GrainInterfaces;
+using Orleans;
+using Orleans.Concurrency;
+using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+using Orleans.Runtime.Host;
+using Orleans.Serialization;
```

Comment:
Please here and every other file, try to remove unnecessary using statements. It's usually harmless, but I found out it made the migration to .net core a lot harder, as some of these are not available.
It would be nice to be able to run these in .net core and compare in the future